strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -377,8 +378,7 @@ def prepare_gaussian_state(self, r, V, modes):
 
         # convert xp-ordering to symmetric ordering
         means = np.vstack([r[:N], r[N:]]).reshape(-1, order="F")
-        C = changebasis(N)
-        cov = C @ V @ C.T
```

Comment:
A part of me will missing seeing this in permutation matrix form -- and always getting confused as to whether it is `C @ V @ C.T` or `C.T @ V @ C` 😆